src/core/index.js

```diff
@@ -149,7 +149,7 @@ if (instanceNamespaces) {
     const executeCommand = executeCommandFactory({
       logger,
       configureCommand,
-      logCommand,
+      debugCommand,
```

Comment:
@jileeshs I don't see this command name change in `executeCommandFactory`.

Reply:
Added. Thanks for noticing it.